src/GraphQL/Resolver/Element.php

```diff
@@ -79,5 +72,151 @@ public function resolveProperties($value = null, $args = [], $context, ResolveIn
+    {
+        $element = $this->getElementById($value['id']);
+        if ($element) {
+            $arguments = $this->composeArguments($args);
+            return $this->extractMultipleElements($element->getChildren(...$arguments), $args, $context, $resolveInfo);
+        }
+        return [];
+    }
+
+    /**
+     * @param null $value
+     * @param array $args
+     * @param $context
+     * @param ResolveInfo|null $resolveInfo
+     * @return array
+     * @throws \Exception
+     */
+    public function resolveSiblings($value = null, $args = [], $context, ResolveInfo $resolveInfo = null)
+    {
+        $element = $this->getElementById($value['id']);
+        if ($element) {
+            $arguments = $this->composeArguments($args);
+            return $this->extractMultipleElements($element->getSiblings(...$arguments), $args, $context, $resolveInfo);
+        }
+        return [];
+    }
+
+    /**
+     * @param array $args
+     * @return array
+     */
+    protected function composeArguments($args = [])
+    {
+        $arguments = [];
+        if ($this->elementType === 'object') {
+            $arguments[] = isset($args['objectTypes']) ? $args['objectTypes'] : [AbstractObject::OBJECT_TYPE_OBJECT, AbstractObject::OBJECT_TYPE_FOLDER];
+        }
+        return $arguments;
+    }
 
+    /**
+     * @param $value
+     * @return AbstractElement|null
+     * @throws \Exception
+     */
+    protected function getElementById($id)
+    {
+        switch ($this->elementType) {
+            case 'asset':
+                return Asset::getById($id);
+            case 'document':
+                return Document::getById($id);
+            case 'object':
+                return AbstractObject::getById($id);
+            default:
+                trigger_error("unknown element type");
+        }
+        return null;
+    }
+
+    /**
+     * @param array $elements
+     * @param array $args
+     * @param $context
+     * @param ResolveInfo|null $resolveInfo
+     * @return array
+     * @throws \Exception
+     */
+    protected function extractMultipleElements($elements, $args, $context, $resolveInfo)
+    {
+        $result = [];
+        if ($elements) {
+            foreach ($elements as $element) {
+                $result[] = $this->extractSingleElement($element, $args, $context, $resolveInfo);
+            }
+        }
+        return array_filter($result);
+    }
+
+    /**
+     * @param Element $element
+     * @param array $args
+     * @param $context
+     * @param ResolveInfo|null $resolveInfo
+     * @return array
+     * @throws \Exception
+     */
+    protected function extractSingleElement($element, $args, $context, $resolveInfo)
+    {
+        if (!WorkspaceHelper::isAllowed($element, $context['configuration'], 'read')) {
+            if (PimcoreDataHubBundle::getNotAllowedPolicy() == PimcoreDataHubBundle::NOT_ALLOWED_POLICY_EXCEPTION) {
```

Comment:
"unknown class PimcoreDataHubBundle" ... import missing ?

Reply:
my bad. will fix it asap